### PR TITLE
package.json update/fix

### PR DESCRIPTION
**Pull request by Klumze**
As we had discussed there was an issue with the main portion of the package.json pointing to a none minified version of the js file even though the prepublish command generated a . min file. I fixed that issue and updated most of the packages being used here. Let me know if there's any issues!

### CODE REVIEW

**Comment by leif81** [comment]
Thanks!

Can you try adding the package-lock.json to the .gitignore as well. It sounds like that file is auto-generated and we can ignore it.

**Comment by Klumze** [comment]
Hey @leif81 just pushed that change to the .gitignore file to my master branch, not sure if I need to create a new merge request or not?

**Comment by leif81** [comment]
@Klumze thank-you, I see that now. No need to open a new pull-request, it'll just add it to this one. One more thing, can you also push a change here to delete the package-lock.json file.

**Comment by Klumze** [comment]
@leif81 There we go haha

File: src/dis/SignalPdu.js [hunk 1]
```diff
@@ -8,116 +8,110 @@
...
+		this.radioId = inputStream.readUShort();
+		this.encodingScheme = inputStream.readUShort();
+		this.tdlType = inputStream.readUShort();
+		this.sampleRate = inputStream.readUInt();
+		this.dataLength = inputStream.readUShort();
+		this.samples = inputStream.readUShort();
+		try {
+			for (var idx = 0; idx < (this.dataLength / 8); idx++) {
+				var anX = new dis.Chunk(1);
+				anX.initFromBinaryDIS(inputStream);
```
Comment (leif81) [inline src/dis/SignalPdu.js]: @Klumze this change to use `initFromBinaryDIS`, remind me what this one was about?

Comment (Klumze) [reply src/dis/SignalPdu.js]: Lol I think this is just cause I had beautified the file to see things better, that change can be discarded if needed

Comment (leif81) [reply src/dis/SignalPdu.js]: Ok yes lets discard that change if it's not required.

**Comment by leif81** [comment]
This has gone a bit stale and has one or more conflicts. I'm going to close because it can't be merged as is.  Please submit new PR if interested z thanks.

**Pull request closed without merging**